### PR TITLE
Remove annoying text-wrap

### PR DESCRIPTION
**Pull request by futa-ikeda**
## Purpose
- Remove annoying text-wrap to preprints metrics disclaimer

## Summary of Changes/Side Effects
- add date of metrics collection start to new line
- 
image

## Ticket

https://openscience.atlassian.net/browse/IN-

## Notes for Reviewer

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] testable and includes test(s)
- [ ] changes described in `CHANGELOG.md`

### CODE REVIEW

File: app/templates/content/index.hbs [hunk 1]
```diff
@@ -107,7 +107,9 @@
                             <div class=" p-v-xs pull-right">
                                 {{t "content.share.views"}}: {{apiMeta.metrics.views}} | {{t "content.share.downloads"}}: {{apiMeta.metrics.downloads}}
                                 {{#bs-tooltip}}
-                                    {{t "content.share.metrics_disclaimer" date=(moment-format this.metricsStartDate "YYYY-MM-DD")}}
+                                    {{t "content.share.metrics_disclaimer"}}
+                                    <br>
```
Comment (brianjgeiger) [inline app/templates/content/index.hbs:111]: Could you use `<br />` here?

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
The tiniest of changes requested. The pickiest of nits.

File: app/templates/content/index.hbs (hunk 1 above)
Comment (futa-ikeda) [reply app/templates/content/index.hbs:111]: Huh, apparently "Self-closing void element is redundant" according to our linter. https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/self-closing-void-elements.md

Comment (brianjgeiger) [reply app/templates/content/index.hbs:111]: Interesting. Well, ignore me, then.

**Review by brianjgeiger — APPROVED** [review]

**Review by adlius — APPROVED** [review]

**Pull request merged**